shared/chat/conversation/messages/dumb.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+
+import React from 'react'
+import Text from './text'
+import {Box} from '../../../common-adapters'
+import {messageStates, followStates} from '../../../constants/chat'
+
+import type {DumbComponentMap} from '../../../constants/types/more'
+
+const baseMock = {
+  author: 'cecileb',
+  message: 'hello world',
+  followState: 'Following',
+  includeHeader: true,
+  onRetry: () => console.log('onRetry'),
+}
+
+const mocks = followStates.reduce((outerAcc, followState) => (
```

Comment:
:sheep: This is pretty hard to read. I think nested for loops could be a lot clearer.